api/app/Models/CommunityExperience.php

```diff
@@ -17,11 +16,11 @@
  * @property string $title
  * @property string $organization
  * @property string $project
- * @property Illuminate\Support\Carbon $start_date
- * @property Illuminate\Support\Carbon $end_date
+ * @property ?\Illuminate\Support\Carbon $start_date
```

Comment:
Pretty sure we should always have this but since we store this as json, we can never be sure so it is safter to mark as nullable